src/compas_fab/backends/ros/backend_features/move_it_inverse_kinematics.py

```diff
@@ -108,12 +111,22 @@ def inverse_kinematics_async(self, callback, errback,
 
         constraints = convert_constraints_to_rosmsg(options.get('constraints'), header)
 
+        timeout_in_secs = options.get('timeout', 2)
+        timeout_duration = Duration(timeout_in_secs, 0).to_data()
+
         ik_request = PositionIKRequest(group_name=group,
                                        robot_state=start_state,
                                        constraints=constraints,
                                        pose_stamped=pose_stamped,
                                        avoid_collisions=options.get('avoid_collisions', True),
-                                       attempts=options.get('attempts', 8))
+                                       attempts=options.get('attempts', 8),
+                                       timeout=timeout_duration)
```

Comment:
Two questions here.  Why the `Duration.to_data()`?  Secondly, the default value for `PositionIKRequest` is a float, which doesn't seem consistent with the ros documentation that says it should be a `Duration`.

Reply:
aha, i see that `Duration` is not a `ROSmsg`, hence the `to_data`

Reply:
good catch, the default `timeout` in `PositionIKRequest` was invalid (I wonder why it worked at all, maybe there's an automatic conversion between float and `Duration`.

Reply:
fixed default on https://github.com/compas-dev/compas_fab/pull/279/commits/8f2fedcb7f227fbcef7fa561c3cbd25a363dd1b9